### PR TITLE
app-crypt/dieharder: Fix cross compile

### PR DESCRIPTION
**Pull request by thughes**

### CODE REVIEW

**Comment by thughes** [comment]
Added signoff

File: app-crypt/dieharder/dieharder-3.31.1-r3.ebuild [hunk 1]
```diff
@@ -7,6 +7,8 @@ DESCRIPTION="An advanced suite for testing the randomness of RNG's"
 HOMEPAGE="http://www.phy.duke.edu/~rgb/General/dieharder.php"
 SRC_URI="http://www.phy.duke.edu/~rgb/General/${PN}/${P}.tgz"
 
+inherit autotools
```
Comment (vapier) [inline app-crypt/dieharder/dieharder-3.31.1-r3.ebuild]: this should be between EAPI & DESCRIPTION above instead

File: app-crypt/dieharder/files/dieharder-3.31.1-cross-compile.patch [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<<<<<<<<<<<<<<<<<<< dieharder-3.31.1-cross-compile.patch >>>>>>>>>>>>>>>>>>>>>>>
```
Comment (vapier) [inline app-crypt/dieharder/files/dieharder-3.31.1-cross-compile.patch]: you don't need this header

File: app-crypt/dieharder/files/dieharder-3.31.1-cross-compile.patch [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+-fi
+-])
+-
+-AC_C_ENDIAN
+- 
++AC_C_BIGENDIAN([AC_SUBST(LITTLE_ENDIAN,0)],[AC_SUBST(LITTLE_ENDIAN,1)])
+ 
+ #==================================================================
+ # Checks for typedefs, structures, and compiler characteristics.
+<<<<<<<<<<<<<<<<<<< dieharder-3.31.1-cross-compile.patch >>>>>>>>>>>>>>>>>>>>>>>
```
Comment (vapier) [inline app-crypt/dieharder/files/dieharder-3.31.1-cross-compile.patch]: you don't need this footer

**Comment by thughes** [comment]
Sent the patch to the list.

**Comment by thughes** [comment]
Ping for merge.

**Pull request closed without merging**